infra/src/dsl/types/input-types.ts

```diff
@@ -149,6 +149,7 @@ export type PersistentVolumeClaim = {
   name?: string
   size: '1Gi' | '5Gi' | '10Gi' | string
   accessModes: AccessModes
+  useExisting?: boolean
```

Comment:
We should maybe discuss if this shouldn't just be a reference to the volume object.

Reply:
the ref() is incomplete, let's iterate on that later.